### PR TITLE
feat: extension structure with refactor

### PR DESCRIPTION
**Pull request by aochmann**
Restructured files and made some refactor (multiline and downloading secure file input type are not working together, moved to secure file with comma separated values).

### CODE REVIEW

File: download-multiple-secure-files-task/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "url": "https://github.com/aochmann/azure-download-multiple-secure-files.git"
+  },
+  "keywords": [
+    "Azure",
+    "VSTS",
+    "DevOps",
+    "Secure Files",
+    "Download Multiple Secure Files"
+  ],
+  "author": "Adrian Ochmann",
```
Comment (mzajkowski) [inline download-multiple-secure-files-task/package.json:25]: Keep it consistent and either use your full name or company name?

**Comment by aochmann** [comment]
Readme will be filled in next PR / release. For now it's for internal usage.

**Comment by aochmann** [comment]
Tests results:
image

File: download-multiple-secure-files-task/task.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      "required": true,
+      "helpMarkDown": "The file names or GUID's of the secure files to download into the agent machine. The files will be deleted after the pipeline runs. Each filename needs to be placed as comma separated values."
+    },
+    {
+      "name": "retryCount",
+      "type": "string",
+      "label": "Retry Count",
+      "defaultValue": "5",
+      "required": false,
+      "helpMarkDown": "Optional number of times to retry downloading a secure file if the download fails."
```
Comment (mzajkowski) [inline download-multiple-secure-files-task/task.json]: I'd reword it to:

Number of times to retry downloading a secure files if the download fails (optional, default: 5).

**Review by mzajkowski — APPROVED** [review]
